tests/unit_tests/dataset_interface_test.py

```diff
@@ -1,13 +1,72 @@
 import unittest
+
 from super_gradients.training.datasets import Cifar10DatasetInterface
+from super_gradients.training.datasets.dataset_interfaces.dataset_interface import PascalVOCUnifiedDetectionDataSetInterfaceV2
+from super_gradients.training.transforms.transforms import DetectionPaddedRescale, DetectionTargetsFormatTransform, DetectionMosaic, DetectionRandomAffine,\
+    DetectionHSV
+from super_gradients.training.utils.detection_utils import DetectionTargetsFormat
+from super_gradients.training.utils.detection_utils import DetectionCollateFN
+from super_gradients.training.utils import sg_model_utils
+from super_gradients.training import utils as core_utils
 
 
 class TestDatasetInterface(unittest.TestCase):
+    def setUp(self) -> None:
+        self.ROOT_DIR = "/home/data/"
+        self.TRAIN_BATCH_SIZE, self.VAL_BATCH_SIZE = 16, 32
```

Comment:
Why all these uppercase letters ?

Reply:
I saw it as constants but actually you're right that it's better without caps.
I changed in d5bc0d4b6a901e2db163f8fa85597a762e1142eb